lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -120,15 +120,8 @@ export type LoggerOptions = {
 
 /**
  *  Use this to configure credential cache preferences in the ClientConfiguration object
- *
- * - claimsBasedCachingEnabled   - Sets whether tokens should be cached based on the claims hash. Default is false.
  */
-export type CacheOptions = {
-    /**
-     * @deprecated claimsBasedCachingEnabled is deprecated and will be removed in the next major version.
-     */
-    claimsBasedCachingEnabled?: boolean;
-};
+export type CacheOptions = {};
```

Comment:
Do we want to remove this altogether since it's empty?

Reply:
+1